python/eight_mile/tf/layers.py

```diff
@@ -938,27 +919,27 @@ def requires_length(self):
         return False
 
 
-def scaled_dot_product_attention(query, key, value, pdrop=0.0, mask=None):
+def scaled_dot_product_attention(query, key, value, mask=None, pdrop=0.0):
```

Comment:
It looks like these attentions aren't doing dropout like the pytorch ones do, this seems like a large mismatch, similarly there is an api mismatch where pytorch takes a dropout object and this is taking a float